common/lib/xmodule/xmodule/x_module.py

```diff
@@ -398,7 +398,7 @@ def get_content_titles(self):
         the display_names of all its leaf nodes.
         """
         if self.has_children:
-            return sum((child.get_content_titles() for child in self.get_children()), [])
+            return sum((child.get_content_titles() for child in self.get_display_items()), [])
```

Comment:
Oh. Bleh. An obvious solution would be to clear `_cached_children` when `bind_for_student` is called. That's not really a great long-term solution, but since I'm hoping the the whole descriptor/module distinction will go away soon, it's probably not awful.

@doctoryes could this be why you had to make the questionable change in the conversion of vertical module?